special-use/special-use-suds-mapping.json

```diff
@@ -6,97 +6,97 @@
       "applicant-info": {
         "type": "object",
         "properties": {
-      		"firstName": { "type": "string", "suds-application": "Contacts", "suds-field": "First Name"},
-      		"lastName": {	"type": "string", "suds-application": "Contact", "suds-field": "Last Name" },
+      		"firstName": { "type": "string", "basic-api-field" : "FIRST_NAME"},
+      		"lastName": {	"type": "string", "basic-api-field" : "LAST_NAME" },
```

Comment:
It's not on the FS form, so I don't think we can ask for it on the intake module. We can put it here just in case? /shrug